Jenkinsfile

```diff
@@ -2,81 +2,96 @@
 
 properties([buildDiscarder(logRotator(artifactDaysToKeepStr: '', artifactNumToKeepStr: '', daysToKeepStr: '14', numToKeepStr: '10')), [$class: 'ScannerJobProperty', doNotScan: false], [$class: 'RebuildSettings', autoRebuild: false, rebuildDisabled: false], pipelineTriggers([])])
 
+pod_template = """
+kind: Pod
+metadata:
+  name: conduce-python-api-build
+spec:
+  containers:
+  - name: build
+    image: python:3.7
```

Comment:
Were you going to try to build both python 2 and 3?  All you'd need to do is add a second container here and copy/paste some of the build/test stages below inside an appropriate container block

Reply:
I just figured I'd get it back to baseline here.  That sounds pretty straightforward though, so maybe I'll take a crack at it after this gets merged.

Reply:
For context, I tried to update the job to use the repos makefile.  I did this because several of the commands in the job did not migrate and could be replaced with make commands. (Make file came after original jenkins job.)